src/Client/Widgets/NumeralRange.fs

```diff
@@ -0,0 +1,60 @@
+﻿module Client.Widgets.NumeralRange
+
+open Browser.Types
+open Fable.Core.JsInterop
+open Fable.React
+open Fable.React.Props
+
+open Client
+open Common.Numerals
+
+let translateNumeralRange = function
```

Comment:
Maybe better `translateToNumeralRange`, or just `toRange`, as we are actually outputting ranges here.